job_templates/readme.md

```diff
@@ -13,8 +13,43 @@ Each job template contains the following informations
 * information card: info.md for display purpose
 * information config: used by program
 
-# Configuration format
+## Configuration format
 
 Configurations are written in HOCON (human optimized object Notation). As a variant of JSON, .conf can also use json format.
 The pyhocon format allows for comments, and you can remove many of the double quotes as well as replace ":" with "=" to make the configurations look cleaner.
 You can find details in [pyhoconb: HOCON Parser for python](https://github.com/chimpler/pyhocon). 
+
+## List of Job Templates
+
+View all the available job templates with the following command:
+
+```nvflare job list_templates```
+
+| Example | Controller-Type | Client Category | Description |
```

Comment:
For the name "Client Category", I change the section "FL Client" -> "Task Execution", because I think our options (Executors VS ModelLearner VS ClientAPI) are really related to how task being executed.

I think the "FL Client" is referring to a larger concept including communication (pull_task, push_results), start worker process (ProcessExecutor), init components and some filters and fire events. 

So suggest changing "Client Category" to maybe "Task Execution Category" or "Task Execution"? Or do you have some other ideas?

Reply:
Changed to "Execution API Type"